tgstation.dme

```diff
@@ -13,7 +13,6 @@
 // END_PREFERENCES
 
 // BEGIN_INCLUDE
-#include "_maps\tgstation2.dm"

```

Comment:
@Jordie0608 so apparently you requested he remove the map from the dme.

@duncathan I think jordie was talking about the actual map file, the dmm file, you had included earlier. THAT shouldn't be included, because this file handles that. BUT this file should stay included.

Your pr had changed it to include both this file, and the dmm map file, this is bad because our server build scripts can't change the map file automatically if you include the dmm.

You should re tick this file, and keep the actual _maps\map_files\TgStation\tgstation.2.1.3.dmm file unticked